### DISCUSSION TITLE
lz4_flex

### DISCUSSION THREAD

**Discussion by roland-5** [ouch-org/ouch#352] [General]
I found today topic about [lz4_flex](https://www.reddit.com/r/rust/comments/10p3gk5/lz4_flex_fast_lz4_decompression_010_released_now/).

I don't know if this is of any use to you. I'm posting this only as a fun fact.

**Comment by figsoda** [comment]
I ran into issues with lz4_flex when tests were initially rewritten: https://github.com/ouch-org/ouch/pull/163, https://github.com/ouch-org/ouch/pull/173. Feel free to try to switch back to lz4_flex again if the original issues have been fixed and if it is benchmarked to be faster than lzzzz.
